system-tests/005/__snapshots__/005.test.js.snap

```diff
@@ -11765,169 +11765,169 @@ Array [
         "text": "Expected line length to be no more than 20 characters (max-line-length)",
       },
       Object {
-        "column": 5,
-        "line": 146,
+        "column": 25,
+        "line": 145,
```

Comment:
One possability is that Jest is _serializing_ the `snapshot.css` file:
https://facebook.github.io/jest/docs/configuration.html#snapshotserializers-array-string

Unconfirmed at this stage though, need to read more docs and or code to confirm.